src/Illuminate/Support/Str.php

```diff
@@ -148,7 +148,7 @@ public static function finish($value, $cap)
      * Determine if a given string matches a given pattern.
      *
      * @param  string|array  $pattern
-     * @param  string  $value
+     * @param  string|int  $value
```

Comment:
It isn't necessary to support an integer, is it? Str class should always interacts with the objects that are exact strings, I think.

Reply:
It's just my opinion. You or any others may add more types to the param docblock if there is a use-case of non-string types. Thank you!